### DISCUSSION TITLE
Exo Gamification Automatic Action for Specific Content / Space

### DISCUSSION THREAD

**Discussion by Amee-r** [exoplatform/exo-community#74] [Q&A]
Hi,

Currently, Gamification in Exo platform provides Automatic points when a user performs a certain action within the exo platform for example, "Like a post".

Here's the existing scenario:
1. Like a post action is created in Program A and Program B.
2. When I like one post, I will get points both for Program A and Program B (Duplicate)  

My question: Is there a way for me to configure, to only reward points to a user when they like a specific post / space? No points will be awarded if the user likes a different post / space. 

Refer below for a scenario example:
Scenario 1 - Specific post
1. User likes "Earth in 2024" post in Space A - Get awarded 50 points.
3. User likes "Moon in 1990" post in Space A - No points are awarded.

Scenario 2 - Perform similar action in different Space (Note that the user is in both Space A and Space B)
1. User likes a post in Space A - Get awarded 50 points.
2. User likes a post in Space B - No points are awarded.

**Comment by plamarque** [comment]
I believe what you're looking for was introduced in [MIP-105](https://github.com/Meeds-io/MIPs/issues/105) in Meeds 7.0.0-M04
![mip-105-like](https://github.com/user-attachments/assets/e971a443-a4dc-4fed-be32-8a219441fa40)

Not yet shipped in exo platform afaik.

**Reply by Amee-r** [reply]
Thanks, @plamarque. This is exactly what I'm looking for. Do you have any idea when it will be available on the Exo platform?
